postscripts/puppet_configure

```diff
@@ -20,10 +20,10 @@
 # Note: If config file is less than 1 hour old, assume it was restored from backup
 #       and do NOT set server.
 #       Contrariwise, if config file is more than 1 hour old, assume this is a new
-#       node and set server according to PUPPET_MASTER env var or $MASTER env var
```

Comment:
A couple comments on why these subtle changes:

- `PUPPET_MASTER` is a typo -- the environment variable comes from the `site` table and should be `PUPPETMASTER`
- `MASTER` is the hostname or IP of the xCAT server. I cannot think of any example where we want to assume the puppet server is the xCAT server. We should always be specifying the puppet server, rather than falling back to assuming it is on the xCAT server (even if it is running on the same server).